### ISSUE TITLE
single quotes not highlighted

### ISSUE THREAD

**Issue by PixnBits** [nathos/sass-textmate-bundle#78]
Might be the same cause of #77 but single quotes are not highlighted the same as double quotes, though [both are valid](http://www.w3.org/TR/2011/REC-CSS2-20110607/syndata.html#strings):

```scss
@import "styles-a";
@import 'styles-b';
```
![image](https://cloud.githubusercontent.com/assets/3945917/7498122/5f3e181a-f3d3-11e4-88e9-f6878ab22dab.png)

**Comment by PixnBits** [comment]
Might be that commit [416b18b41](https://github.com/nathos/sass-textmate-bundle/commit/416b18b411a3f65090ca7a2f90bfc5840127d52e#diff-2fbb7377762698486d25359cb90fbe4cR130) was wonky (if even just whitespace)?

**Comment by HugoGiraudel** [comment]
Any chance we could get this fixed? :)

**Comment by defaye** [comment]
Would like this to be fixed too.

**Comment by callumlocke** [comment]
+1

**Comment by guerrero** [comment]
I've created a pull request to solve this problem. Until it's merged you can follow this steps to use a fixed version:

1. Remove your current package: `Command Palette` > `Package Control: Remove Package` > `Sass`
2. Copy source code from the fixed branch in your `Packages` folder:
```bash
wget -P "$HOME/Library/Application Support/Sublime Text 3/Packages" "https://codeload.github.com/guerrero/sass-textmate-bundle/zip/sublime"
```
You can copy it manually downloading the compressed package from [here][1]

[1]: https://codeload.github.com/guerrero/sass-textmate-bundle/zip/sublime

**Comment by PixnBits** [comment]
Nice!

**Comment by HugoGiraudel** [comment]
Ping @guerrero: anything specific to do after the `wget` command? I can't seem to see the package. :(

**Comment by guerrero** [comment]
Sorry @HugoGiraudel, I've updated my previous message with a working script explained

**Comment by PixnBits** [comment]
ping @nathos

**Comment by nathos** [comment]
@PixnBits Thanks :)

Merged in #79 to fix this
